torch/csrc/jit/codegen/cuda/ir_utils.h

```diff
@@ -355,6 +357,10 @@ TORCH_CUDA_CU_API bool isIndexSelectLookupTv(const TensorView* tv);
 // Check if the given tv is third argment of index_select(lookup, dim, indices)
 TORCH_CUDA_CU_API bool isIndexSelectIndicesTv(const TensorView* tv);
 
+// Check if the given tv is first/third argment of torch_gather(lookup, dim, indices)
```

Comment:
If `first/third`, then why in the above `isTorchGatherIndicesTv/isTorchGatherLookupTv`, we use `input(0)/input(1)` rather than `input(0)/input(2)`?

Reply:
The API interface is torch.gather(input_tv, dim, index_tv). So, the first input is input_tv, second is dim, and third is index_tv. And in these three parameters, only input_tv and index_tv is Tensor (TensorView), the dim is only an Integer. And the input() method can only get tensor parameters. So input(0) is input_tv, input(1) is index_tv and input(2) does not make sense.